python/cudf/cudf/tests/test_orc.py

```diff
@@ -681,7 +681,6 @@ def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
 def test_orc_chunked_write_statistics(tmpdir, datadir, nrows, stats_freq):
     from pyarrow import orc
 
-    np.random.seed(0)
     supported_stat_types = supported_numpy_dtypes + ["str"]
```

Comment:
*question*: Does `gen_rand_series` now arrange for a seed to be set?

Reply:
yes, seed is now a pass-through param via `gen_rand_series`.